jar-infer/src/main/java/com/uber/nullaway/jarinfer/definitelyDerefedParams.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * **************************************************************************** Copyright (c) 2018
+ * Uber Technologies Inc.
+ *
+ * <p>Contributors: Uber Technologies Inc.
+ * ****************************************************************************
+ */
+package com.uber.nullaway.jarinfer;
+
+import com.ibm.wala.cfg.ControlFlowGraph;
+import com.ibm.wala.classLoader.IMethod;
+import com.ibm.wala.ipa.cfg.ExceptionPrunedCFG;
+import com.ibm.wala.ipa.cfg.PrunedCFG;
+import com.ibm.wala.ipa.cha.IClassHierarchy;
+import com.ibm.wala.ssa.IR;
+import com.ibm.wala.ssa.ISSABasicBlock;
+import com.ibm.wala.ssa.SSAAbstractInvokeInstruction;
+import com.ibm.wala.ssa.SSAGetInstruction;
+import com.ibm.wala.ssa.SSAInstruction;
+import com.ibm.wala.ssa.SSAPutInstruction;
+import com.ibm.wala.util.collections.Iterator2Iterable;
+import com.ibm.wala.util.graph.Graph;
+import com.ibm.wala.util.graph.GraphUtil;
+import com.ibm.wala.util.graph.dominators.DominanceFrontiers;
+import com.ibm.wala.util.graph.impl.GraphInverter;
+import java.util.*;
+
+/*
+ * Identify definitely-dereferenced function parameters
+ * v0.1
+ * Basic analysis that identifies function parameter dereferences in BBs that post-dominate the exit node.
+ *
+ * @author subarno
```

Comment:
We don't usually do `@author` tags. Nothing wrong with them, but this should already be visible from `git blame`, etc :)